woocommerce/woocommerce
# Add product page spotlight tour

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32421.


Acceptance criteria:

- [x] New walkthrough available for users that select the physical product template
- [x] Enable the spotlight for each step
- [x] Replace the "Add product" button right close to the title "Edit product" with the "Enable guided mode" button.

To be done in follow-up PRs:
- The next button is disabled till the action is completed.
- ~Enable the focus state to nudge the user to interact and take action~

### How to test the changes in this Pull Request:

1. Run `pnpm install`
2. Run `pnpm nx start woocommerce-admin`
3. Enable `experimental-product-tour` feature flag via WooCommerce Admin Test Helper
4. Go to `Woocommerce > Home`
5. Click `Add my products` task
6.  Select "Start with a template"
7. Select "Physical product"
8. Confirm spotlight tour is shown with the following features.

- The number of steps is always visible.
- The number of steps is reduced to 8.
- The page scrolls up and down automatically based on the information highlighted.

9. Complete the step tour
10. Click the "Enable guided mode" button
11. Observe that the tour is shown again.
12. Create another product with another product type
13. Observe that the "Enable guided mode" button is NOT displayed.

design: cJQeyyhx9PraQrY3YAtCGM-fi-1104%3A127916

### Other information:




### FOR PR REVIEWER ONLY:
